spec/schemas/ml._common.yaml

```diff
@@ -187,6 +187,160 @@ components:
+          items:
+            $ref: '#/components/schemas/InferenceResults'
+        status:
+          type: string
+          description: The status.
+        prediction_result:
+          $ref: '#/components/schemas/PredictionResult'
+    InferenceResults:
+      type: object
+      properties:
+        output:
+          type: array
+          items:
+            $ref: '#/components/schemas/Output'
+    Output:
+      type: object
+      properties:
+        name:
+          type: string
+          description: The output name.
+        data_type:
+          type: string
+          description: The output data type.
+          enum:
+            - BOOLEAN
+            - FLOAT16
+            - FLOAT32
+            - FLOAT64
+            - INT32
+            - INT64
+            - INT8
+            - STRING
+            - UINT8
+            - UNKNOWN
+        shape:
+          type: array
+          items:
+            type: integer
+            format: int64
+            description: The output shape.
+        data:
+          type: array
+          items:
+            type: number
+            format: double
+            description: The output data.
+        byte_buffer:
+          $ref: '#/components/schemas/ByteBuffer'
+      required:
+        - data
+    ByteBuffer:
+      type: object
+      properties:
+        array:
+          type: string
+          description: The byte buffer array.
+        order:
+          type: string
+          description: The byte buffer order.
+          enum:
+            - BIG_ENDIAN
+            - LITTLE_ENDIAN
+    PredictionResult:
+      type: object
+      properties:
+        column_metas:
+          type: array
+          items:
+            $ref: '#/components/schemas/ColumnMeta'
+        rows:
+          type: array
+          items:
+            $ref: '#/components/schemas/Rows'
+    ColumnMeta:
+      type: object
+      properties:
+        name:
+          $ref: '_common.yaml#/components/schemas/Name'
+        column_type:
+          type: string
+          description: The column type.
+          enum:
+            - BOOLEAN
+            - DOUBLE
+            - INTEGER
+            - STRING
+    Rows:
+      type: object
+      properties:
+        values:
+          type: array
+          items:
+            $ref: '#/components/schemas/Values'
+    Values:
+      type: object
+      properties:
+        column_type:
+          type: string
+          description: The column type.
+          enum:
+            - BOOLEAN
+            - DOUBLE
+            - INTEGER
+            - STRING
+        value:
+          type: integer
+          format: int64
+          description: The value.
+    InputQuery:
+      type: object
+      properties:
+        _source:
+          type: array
+          items:
+            type: string
+          description: The source fields
+        size:
+          type: integer
+          format: int64
+          description: The size of the query.
```

Comment:
I dont know exactly the format of size
If I understood correctly, you can use this. But you can leave it as it is if I misunderstood.

https://github.com/opensearch-project/opensearch-api-specification/blob/edf9d6f0dbf1068345b58cb9e2c3464fa9839226/spec/schemas/_common.yaml#L832

Reply:
I believe in this context, size refers to the max number of documents to return in the query result, not the byte count of data. So, we can leave it as it is.